docker/pudl_etl.sh

```diff
@@ -5,9 +5,13 @@ pudl_setup \
     --pudl_out $CONTAINER_PUDL_OUT
 ferc1_to_sqlite \
     --clobber \
+    --gcs-cache-path gs://zenodo-cache.catalyst.coop \
+    --bypass-local-cache \
```

Comment:
Did you get a sense of how much of a difference in performance pulling from the storage buckets rather than local disk made?

Reply:
No, I haven't profiled it yet.